tailcfg/tailcfg.go

```diff
@@ -1797,8 +1798,26 @@ type MapResponse struct {
 	// Note that this package's type, due its use of a slice and omitempty, is
 	// unable to marshal a zero-length non-nil slice. The control server needs
 	// to marshal this type using a separate type. See MapResponse docs.
+	//
+	// See PacketFilters for the newer way to send PacketFilter updates.
 	PacketFilter []FilterRule `json:",omitempty"`
 
+	// PacketFilters is a method to send incremental packet filter updates to
```

Comment:
"is a method" confused me at first because I thought it meant in the Go sense. Perhaps "encodes incremental packet filter updates" or words to that effect?

Reply:
done